tests/unit/test_analytics.py

```diff
@@ -70,7 +70,7 @@ def test_send(mock_post, tmp_path):
 
     analytics.send(str(report_file))
     assert mock_post.called
-    assert mock_post.call_args.args[0] == url
+    assert mock_post.call_args[0][0] == url
```

Comment:
We were using `mock` before which backports all features from `3.8`. `call.args` is only available on 3.8. 

On this change, it's not necessary as we can do some via `[0][0]`.